content/en/blog/2023/outreachy-how-why-sakshi.md

```diff
@@ -0,0 +1,74 @@
+student at RGPV in the heart of India. My journey into the tech universe has
+been fueled by a blend of curiosity, the pursuit of honesty, and an unwavering
+commitment to perseverance. These are the pillars that not only uphold my
+personal values but also form the foundation of my professional aspirations.
+
+## The Spark of Curiosity
+
+From a young age, I was mesmerized by the boundless potential that technology
+promised. It was a combination of curiosity and the allure of creating something
+out of nothing that led me to dive headfirst into the world of open-source.
+Within the past year, I’ve had the chance to contribute to diverse projects,
+where I learned that every line of code could be a small step towards making a
+significant impact.
+
+## The Pursuit of Honesty
+
+Honesty, for me, translates into transparency and openness, especially in the
+realm of software development. It’s about admitting what we don’t know, seeking
+help when needed, and being candid about our capabilities. It’s honesty that has
+allowed me to embrace my past as an Outreachy applicant with all its lessons,
+and it’s honesty that drives me to share my learnings and experiences with the
+community.
+
+## The Power of Perseverance
+
+Challenges are an inevitable part of life, more so in a field as dynamic as
+technology. During my academic tenure, I grappled with concepts, whether it was
+establishing CI/CD pipelines or developing Golang projects like a banking app
+with REST APIs, and gRPC services for my college portal. These challenges tested
+my resolve but also reinforced my belief in perseverance. It's the commitment to
+keep going, to learn and to improve, that ultimately transforms setbacks into
+successes.
+
+## The Motivation Behind Outreachy
+
+Outreachy for me was not just a program; it represented the intersection of my
+core values with an opportunity for growth. My motivation to apply stemmed from
+a drive to tap into a global network of like-minded individuals and mentors
+across the OpenTelemetry community and the broader open-source ecosystem.
+
+My previous application to Outreachy was a learning curve, a process through
+which I discovered areas for improvement and gained insight into the
+collaborative nature of open-source projects. What attracted me to reapply were
+the very tenets of Outreachy—diversity, inclusivity, and empowerment.
+
+Outreachy offers a nurturing ground to enhance skill sets, build confidence, and
+foster community engagement, which aligns perfectly with my aspiration to be a
+valuable contributor to the OTel community while embodying my core values.
+
+## Why OpenTelemetry?
+
+OpenTelemetry (OTel) stood out to me as a space where security and scalability
+are not just focused on but are thriving. As an intern with the SIG Security
+team, I’ll be championing the implementation of organization-wide repository
+governance policies and checks. This isn’t just about enforcing standards; it’s
```

Comment:
```suggestion
team, I’ll be championing the implementation of organization-wide repository
governance policies and checks. This isn’t just about enforcing standards; it’s
```

is there a GitHub issue that you can link from here? Do you want to mention AllStar somewhere?